locked-nft/transactions/lock_nft.cdc

```diff
@@ -0,0 +1,22 @@
+import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
+import NFTLocker from "../contracts/NFTLocker.cdc"
+import ExampleNFT from 0xEXAMPLENFTADDRESS
+
+
+transaction(nftID: UInt64, duration: UInt64) {
```

Comment:
I think locking should also trigger a storefront removeListing event like we do for nfl burning: https://github.com/dapperlabs/nfl-app/blob/master/src/edge/flow/cadence/transactions/challenges/burnNFTs.cdc#L29

Reply:
Yes, I think so too. We can do that in the lock tx. For the moment, EPL does not have a storefront, so will add to tx when we launch this feature. Good suggestion.